### PR TITLE
Addendum

### PR DESCRIPTION
**Pull request by ElDeveloper**
I've added the HTML formatted version of the Addendum as authored by @JWDebelius. This is likely not ready to be merged. I've made it available via the [following page](http://webdev.microbio.me/americangut/addendum.psp), to get to this page users will have to login with their account and click in the left sidebar "The American Gut Addendum", which should open a new page with all the information that's currently contained in this document.

@JWDebelius, would you mind taking a look at it and making sure everything is looking good?

### CODE REVIEW

File: www/americangut/addendum.psp [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+                <li><a href="index.psp"><img class="icon" src="/qiime/img/login_icon.png">Participant Login</a></li>
+                </ul>
+            </div>
+            <div class="content">
+                <div class="faq_wrapper">
+                    <div class="faq">
+                        <div id = 'text' align="justify">
+                            <a name = "top"><h1>American Gut Addendum</h1></a>
+                            <p>First, we&apos;d like to note that in general these data allow you to understand how similar or different you are to other people in terms of the bacterial composition of the sample you sent. The information about the microbes is at as fine level of a taxonomic resolution as we were able to achieve with our sequencing methods, and varies for different groups of microbes. Currently, we cannot tell you what it means if you have more or less of a certain bacteria than other people. Gut microbiome research is still new, and we have a lot to learn. Your participation in the American Gut Project will allow us to learn more, and we hope to update you with new findings as they emerge.</p>
```
Comment (teravest) [inline www/americangut/addendum.psp]: Take out First we'd like to note that in General.  start with These data...

File: www/americangut/addendum.psp [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+                            </ul>
+                            <h5>Chimpanzee</h5>
+                            <ul>
+                                <li>Kingdom: Animalia</li>
+                                <li>Phylum: Chordata</li>
+                                <li>Class: Mammalia</li>
+                                <li>Order: Primates</li>
+                                <li>Family: Hominidae</li>
+                                <li>Genus: <em>Pan</em></li>
+                                <li>Species: <em>troglodyte</em></li>
```
Comment (teravest) [inline www/americangut/addendum.psp]: should be troglodytes

File: www/americangut/addendum.psp [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+                            </tr>
+                            <tr>
+                                <td class = 'taxa'>Genius <em>Clostridium</em></td>
+                                <td>0.9%</td>
+                                <td>0.1%</td>
+                                <td>8x</td>
+                            </tr>
+                        </table>
+
+                        <p> The second table shows four microbes that you had more of compared to other people. It is likely that other participants also have these microbes in their sample, but we found substantially higher abundances of them in your sample relative to others. The percentages on the right tell you how many of your total bacteria (under &quot;Sample&quot;) or of the total bacteria in an average person&amp;s sample (under &quot;Population&quot;) belong to these taxa. Since you have more of these bacteria than most other people, the percentage under &quot;Sample&quot; should be higher than the percentage under &quot;Population&quot;.</p>
```
Comment (teravest) [inline www/americangut/addendum.psp:386]: this has "person&s" instead of "person's"

File: www/americangut/addendum.psp [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+
+                        <p>The fold change tells you how many more of these bacteria you have than the average participant. For example, if you have 20% Bacteria A and the average person in the population has 10% Bacteria A, you have twice as many Bacteria A. This would be a twofold (2x) difference. Please note that because the percentages we report on this sheet are rounded (e.g., 0.05% rounded to 0.1%), and your fold differences are calculated from values that are not rounded, the fold differences you see may be slightly distinct than what you would calculate based on the numbers you see.</p>
+
+                        <h3>Rare Taxa</h3>
+
+                        <h6>This sample included the following rare taxa: Fenus <em>Varibaculum</em>, Genus <em>Neisseria</em>, Genus <em>Campylobacter</em>, Unclassified Order ML615J-28.</h6>
```
Comment (teravest) [inline www/americangut/addendum.psp]: the font on this line is way too small

File: www/americangut/addendum.psp [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+
+                        <h3>Different Body Sites</h3>
+                        <img src = "img/PCoA1.png"
+                             alt = "PCoA by body site for AGP and HMP"/>
+                        <p>This plot lets you compare your sample to samples collected by two microbiome projects from several body sites. The color of each point tells you which project and body site the sample came from. HMP refers to the <a href = 'http://www.hmpdacc.org'>Human Microbiome Project</a>, funded by the National Institutes of Health. You can see how your sample compared to fecal, oral, and skin samples from the Human Microbiome Project, as well as to fecal, oral, and skin samples from the American Gut Project, the Global Gut Project, and the Personal Genome Project. These samples have been combined in any category not labeled &quot;HMP&quot;. The oval around each group of points shows you where an average sample from each project and body site should fall on the plot. These sometimes make it easier to see the patterns all the clusters of points make.</p>
```
Comment (teravest) [inline www/americangut/addendum.psp]: should be other microbiome projects instead of two microbiome progjects

File: www/americangut/addendum.psp [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+
+                        <h4>Verrucomicrobia</h4>
+                        <p>A relatively new phylum with only a handful of described species. Although not the most abundant, they seem to be always present in soil, aquatic environments, and feces. Most likely they are involved in the decomposition of organic matter, with no known pathogens. While some may be autotrophs, others can be internal symbionts of microscopic eukaryotes such as protists or nematodes. Their name is derived from a wart-like appearance (<em>verruca</em> means wart) but they do not cause warts.</p>
+
+                        <h4>Tenericutes</h4>
+                        <p>A phylum of Gram-negative bacteria without a cell wall (<em>tener</em> - soft, <em>cutis</em> - skin) which are Organized in a single class. Nutritionally, they represent variable pathways ranging from aerobic and anaerobic fermenters to commensals to strict pathogens of vertebrates (e.g., fish, cattle, wildlife). Among the best studied are Mycoplasmas with a fried egg-like shape and <em>Mycoplasma pneumoniae</em> is one of the best known examples of human pathogens causing pneumonia, bronchitis, and other respiratory conditions.</p>
```
Comment (teravest) [inline www/americangut/addendum.psp]: Organized shouldn't be capatilzed

**Comment by ElDeveloper** [comment]
@JWDebelius are all corrections done now?

**Comment by jwdebelius** [comment]
No, I still have to implement the suggestions Meg and Dorota made.

**Comment by ElDeveloper** [comment]
Cool. Let me know.

Thanks!

Yoshiki.

**Comment by ElDeveloper** [comment]
Awesome, thank @JWDebelius this is looking freaking amazing!

**Comment by wasade** [comment]
Can this be merged?

**Comment by jwdebelius** [comment]
I have not heard anything new from anyone, so I would say yes.

**Pull request merged**